Carbon.Windows/Functions/Remove-CEventLog.ps1

```diff
@@ -0,0 +1,34 @@
+
+function Remove-CEventLog
+{
+    <#
+    .SYNOPSIS
+    Removes an event log.
+
+    .DESCRIPTION
+    The `Remove-CEventLog` function removes an event log from the local computer.
+
+    This function will write an error if the event log does not already exist.
+
+    .EXAMPLE
+    Remove-CEventLog -LogName 'TestApp'
+
+    Demonstrates removing the `TestApp` event log.
+    #>
+    [CmdletBinding()]
+    param(
+        # The name of the log to remove.
+        [Parameter(Mandatory)]
+        [String] $LogName
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
+
+    if ([Diagnostics.EventLog]::GetEventLogs().LogDisplayName -like $LogName)

```

Comment:
Use `-eq` instead of `-like` since the `Delete` method expects the actual log name. (I'm assuming. Maybe it excepts wildcards?)